.github/workflows/ci.yml

```diff
@@ -25,11 +26,8 @@ jobs:
         run: |
           python -m pip install --upgrade pip
           pip install -e ".[dev]"
-          sudo apt-get install graphviz
-      - name: Run tests for Python ${{ matrix.python-version }}
-        run: |
-          # Constructing Tox environment names from Github Python version strings should become unnecessary once/if this ticket gets implemented: https://github.com/tox-dev/tox/issues/2115
-          PYVER=${{matrix.python-version}}; TOXENV="py"${PYVER//.}
-          tox -e "${TOXENV}"
+          python -m pip install tox-gh-actions
+      - name: Run tests
+        run: tox
       - name: Test docs can be built
         run: tox -e docs
```

Comment:
Buildings docs is independent on platform and python version right?
Maybe should be its own job

Reply:
For posterity: talked afk, decided to keep it for now, and move to a separate job later when windows runner support is enabled for all other IQM repositories.